source/_components/binary_sensor.owlet.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Owlet Binary Sensor"
+description: "Instructions on how to integrate Owlet baby monitor into Home Assistant."
+date: 2019-01-19
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: owlet.svg
+ha_category: Health
+featured: true
+ha_release: "0.87"
+ha_iot_class: "Cloud Polling"
+---
```

Comment:
We can merge this page with the component page as platforms don't require any configuration.